src/main/java/de/tum/in/www1/artemis/domain/Achievement.java

```diff
@@ -0,0 +1,166 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import javax.persistence.*;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.enumeration.AchievementRank;
+import de.tum.in.www1.artemis.domain.enumeration.AchievementType;
+
+@Entity
+@Table(name = "achievement")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Achievement implements Serializable {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "title")
+    private String title;
+
+    @Column(name = "description")
+    private String description;
+
+    @Column(name = "icon")
+    private String icon;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "`rank`")
+    private AchievementRank rank;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "`type`")
```

Comment:
I would suggest not using SQL keywords for column names. https://www.drupal.org/node/141051

Reply:
same as above